extensions/wikia/Qualaroo/scripts/Qualaroo.js

```diff
@@ -70,5 +80,21 @@
 		createCookie('qualaroo_survey_submission');
 	});
 
+	if (window.ads.context.opts.sourcePointDetection) {
+		if (!window.ads.runtime || !window.ads.runtime.sp || window.ads.runtime.sp.blocking === undefined) {
```

Comment:
per discussion in the corridor, we can add a comment that this is the same logic as in: https://github.com/Wikia/app/blob/dev/extensions/wikia/ARecoveryEngine/js/sourcePointHelper.js#L54